internal/config/config_test.go

```diff
@@ -16,12 +17,27 @@ func TestConfig(t *testing.T) {
 	if err != nil {
 		t.Fatalf("GetConfigurationData failed with err %v", err)
 	}
-	err = cfg.ParseConfiguration(configData)
+	err = cfg.ParseConfiguration(&config.ConfigData{}, configData)
```

Comment:
would be nicer if ParseConfiguration would return the configdata object, instead of being a parameter... think of more functional programming